src/libMetrics/internal/mixins.h

```diff
@@ -124,84 +114,91 @@ class DoubleHistogram {
 
 class DoubleGauge {
  public:
-  DoubleGauge(const std::string &name, const std::string &description,
-              const std::string &units, bool obs)
-      : m_theGauge(Metrics::GetInstance().CreateDoubleGauge(
-            zil::metrics::GetFullName(METRIC_FAMILY, name), description,
-            units)) {}
+  DoubleGauge(zil::metrics::FilterClass fc, const std::string &name, const std::string &description, const std::string &units,
+              bool)
```

Comment:
Is this bool neeeded?

Reply:
There might be a better way, is simply delineates ambiguity between the other templates, it is behind the scenes but helps the compiler generate the correct code for the type as there are three possibilities it really signifies does this routine have a callback (Asynchronous or Synchronous) as it is never tested we removed the name so the compiler doesn't complain about the unused variable. @Artem could probably change this if it's offensive.